src/HW2/divide.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+
+int main()
+{
+    int a;
+    int b;
```

Comment:
Вообще, здесь тодже надо инициализировать.
```suggestion
    int a = 0;
    int b = 0;
```